room.html

```diff
@@ -2,11 +2,21 @@
 <html>
 <head>
     <script src='https://meet.jit.si/external_api.js'></script>
-	<script type="text/javascript" charset="utf-8" src="assets/js/reconnecting-websocket.min.js"></script>
+    <script type="text/javascript" charset="utf-8" src="assets/js/reconnecting-websocket.min.js"></script>
+    <script src="assets/js/feedback.js"></script>
     <link rel="stylesheet" type="text/css" href="assets/css/room.css">
+    <link rel="stylesheet" type="text/css" href="assets/css/feedback.css">
 </head>
 <body>
     <div id="meet"/>
+
+    <div id="extras">
+        <div class="tool" onclick="feedback('🐇')">🐇</div>
```

Comment:
for me (local deployment, running `/usr/bin/python3 -m http.server 8081 `) all those show up as
![DeepinScreenshot_select-area_20200628123704](https://user-images.githubusercontent.com/39889/85953100-1a619300-b93c-11ea-8fcd-6f39bfde5c34.png)

Reply:
For the icons not showing up in local development, maybe copy/pasting the emojis could be replaced by explicitly typing their unicode codes? like `console.log("\u{1F430}")` (I used this handy website for the conversion: https://r12a.github.io/app-conversion/)

Reply:
```html
<div class="tool" onclick="feedback(`${"\u{1F430}"}`)">&#x1F430;</div>
```

Reply:
what's your browser? does it display the icons in the brain-web jisti room here? https://brain-web.github.io/videoconference/

Reply:
Chromium
Version 79.0.3945.130 (Developer Build) built on Debian bullseye/sid, running on Debian bullseye/sid (64-bit)

https://brain-web.github.io/videoconference/ works ok, it is only when I tried this branch locally (may be should try brain-web. locally?)

Reply:
I wonder if it just needs some html or css setup for those to appear correctly, and which was not copied over from brain-web?

Reply:
e.g. they (obscure characters) are black not white like on brain-web

Reply:
BTW 
- do they work for you when you test them locally on your laptop? (just try `/usr/bin/python3 -m http.server 8081` and go to localhost:8081 ;))
- they appear incorrect in "inspect" view for this page, but correctly in the one for brainweb

Reply:
also i see
```
Uncaught ReferenceError: api is not defined
    at feedback.js:29
```
in errors within inspect -- I guess smth to fix

Reply:
ha! funny... with the python server I get the currency icons instead of the emojis, like you
(normally I use the `http-server` I installed with `npm`)

Reply:
the python server wanted to be told specifically the encoding to use. Adding this to room.html solved the problem:

```html
<meta content="text/html;charset=utf-8" http-equiv="Content-Type">
<meta content="utf-8" http-equiv="encoding">
```

Reply:
I also made a function to install the event listener for the data channel instead of having wild code in feedback.js